py/htm/encoders/eye.py

```diff
@@ -427,55 +440,49 @@ def compute(self, position=None, rotation=None, scale=None):
         if scale is not None:
           self.scale=scale
 
-        # apply field of view (FOV)
+        # apply field of view (FOV) & rotation
+        self.image = Eye.new_image_(image) #TODO remove the FOV, already done in retina's logPolar transform
+        self.image = self.rotate_(self.image, rotation)
         self.roi = self._crop_roi()
 
         # Retina image transforms (Parvo & Magnocellular).
         self.retina.run(self.roi)
+        
         if self.parvo_enc is not None:
           parvo = self.retina.getParvo()
         if self.magno_enc is not None:
           magno = self.retina.getMagno()
 
         # Log Polar Transform.
-        center = self.retina_diameter / 2
-        M      = self.retina_diameter * self.fovea_scale
+        center = self.retina.getInputSize()[0] / 2
+        M      = self.retina.getInputSize()[0] * self.scale
         if self.parvo_enc is not None:
           parvo = cv2.logPolar(parvo,
```

Comment:
should we apply the logPolar only in the Channel encoder? Bcs that one expects its data in that form.